studies/models.py

```diff
@@ -51,8 +54,10 @@ class Study(models.Model):
     criteria = models.TextField()
     duration = models.TextField()
     contact_info = models.TextField()
-    max_age = models.TextField(default='')
-    min_age = models.TextField(default='')
+    min_age_months = models.IntegerField(null=True, default=0, choices=MONTH_CHOICES)
```

Comment:
`null=True` and a `default` value doesn't **really** make any sense unless you really want existing rows to stay `null` and new rows to use the default.